MobileCenterCrashes/MobileCenterCrashesTests/MSCrashesTests.mm

```diff
@@ -149,6 +160,21 @@ - (void)testDelegateMethodsAreCalled {
   OCMVerify([delegateMock crashes:[MSCrashes sharedInstance] didFailSendingErrorReport:errorReport withError:nil]);
 }
 
+- (void)testCrashHandlerSetupDelegateMethodsAreCalled {
+
+  // If
```

Comment:
@TroubleMakerBen In your PR: https://github.com/Microsoft/mobile-center-sdk-ios/pull/684/files#diff-7774e2ea056e6d362b3859e0ff8d5355